File: JS/FizzBuzz/fizzbuzz.js
const fizzBuzz = (number) => {
    const multipleOf3 = (number) => {
        return number % 3 === 0;
      };
      
      const multipleOf5 = (number) => {
        return number % 5 === 0;
      };

    if (multipleOf3(number) && multipleOf5(number)) return "FizzBuzz";
    else if (multipleOf3(number)) return "Fizz";
    else if (multipleOf5(number)) return "Buzz";
  
    return number;
};
  
module.exports = fizzBuzz;

File: JAVA/FizzBuzzTDD/src/main/java/org/example/FizzBuzz.java
package org.example;

public class FizzBuzz {
    public String convertNumber(int number){
        String value = "";

        if(number % 3 == 0 && number % 5 == 0)
            value = "FizzBuzz";
        else if (number % 5 == 0)
            value = "Buzz";
        else if(number % 3 == 0)
            value = "Fizz";
        else
            value = Integer.toString(number);

        return value;
    }
}
